src/mono/wasm/runtime/types-api.ts

```diff
@@ -0,0 +1,245 @@
+import type { EmscriptenModule, NativePointer } from "./types/emscripten";
+
+export interface DotnetHostBuilder {
```

Comment:
I don't see `withStartupMemoryCache`, it that correct?

Reply:
Yes, that's not public API